Src/Support/GoogleApis.Core/NetStandard/project.json

```diff
@@ -1,23 +1,9 @@
 ﻿{
   "supports": {},
   "dependencies": {
-    "Newtonsoft.Json": "9.0.1",
-    "System.Collections": "4.0.11",
-    "System.Console": "4.0.0",
-    "System.Diagnostics.Debug": "4.0.11",
-    "System.Globalization": "4.0.11",
-    "System.IO": "4.1.0",
-    "System.Linq": "4.1.0",
-    "System.Net.Http": "4.1.0",
-    "System.Net.Primitives": "4.0.11",
-    "System.Reflection": "4.1.0",
-    "System.Reflection.Extensions": "4.0.1",
-    "System.Reflection.TypeExtensions": "4.1.0",
-    "System.Runtime": "4.1.0",
-    "System.Runtime.Extensions": "4.1.0",
-    "System.Text.RegularExpressions": "4.1.0",
-    "System.Threading": "4.0.11",
-    "System.Threading.Tasks": "4.0.11"
+    "NETStandard.Library": "1.6.1",
+    "System.Reflection.TypeExtensions": "4.3.0",
+    "Newtonsoft.Json": "9.0.1"
```

Comment:
I think so.
google-api-dotnet-client netcore is on v9
google-api-dotnet-client <everything else> is on v7
We need to use v10 in Gax, for which I'm now preparing a PR.
google-api-dotnet-client needs to upgrade all platforms to use v10, but I'm loath to do that in this release because it's a change that *may* cause some trouble, and there are a few other such changes that may be required soon in google-api-dotnet-client. I'd rather release them all at the same time, to minimize the number of potentially troublesome releases.
Do you think this is reasonable? Or should this PR upgrade all platforms to v10?